File: frontend/portfolioERP-web/src/app/features/suppliers/models/create-supplier-request.ts
export interface CreateSupplierRequest {
  supplierCode: string;
  companyName: string;
  contactName: string | null;
  vatNumber: string | null;
  taxCode: string | null;
  email: string;
  phone: string | null;
  address: string | null;
  city: string | null;
  province: string | null;
  postalCode: string | null;
  country: string;
}

File: backend/PortfolioERP.Application/Features/Suppliers/CreateSupplierRequest.cs
namespace PortfolioERP.Application.Features.Suppliers;

public sealed record CreateSupplierRequest(
    string SupplierCode,
    string CompanyName,
    string? ContactName,
    string? VatNumber,
    string? TaxCode,
    string Email,
    string? Phone,
    string? Address,
    string? City,
    string? Province,
    string? PostalCode,
    string Country);